releases/Process/Major Release Process.md

```diff
@@ -70,16 +70,31 @@ Entry Criteria: Release SIG has has indicated the need for a release and has a g
 1. **Release SIG** coordinates with the Technical Steering Committee, the Open 3D Foundation Marketing Committee, and the SIGs to determine a release date. This may be done by the SIG chair, co-chair, or the release manager if one has been chosen.
```

Comment:
In which channel did the Release SIG discuss this? In the SIG release Discord channel?

Reply:
good catch. I've updated to include the Discord channels